pandas/core/index.py

```diff
@@ -1083,22 +1083,18 @@ def identical(self, other):
     def asof(self, label):
         """
         For a sorted index, return the most recent label up to and including
-        the passed label. Return NaN if not found
-        """
-        if isinstance(label, (Index, ABCSeries, np.ndarray)):
-            raise TypeError('%s' % type(label))
-
-        if not isinstance(label, Timestamp):
-            label = Timestamp(label)
-
-        if label not in self:
-            loc = self.searchsorted(label, side='left')
-            if loc > 0:
-                return self[loc - 1]
-            else:
-                return np.nan
+        the passed label. Return NaN if not found.
 
-        return label
+        See also: get_loc_nearest
```

Comment:
can you make this a `See also` section?

```
See also
--------
get_loc_nearest : small explanation what this is
```

then you get automatic linking in the html pages